vce-two-trains-alight-and-board.py

```diff
@@ -625,11 +625,20 @@ def make_chart_2(
     return wb
 
 
+def run_model(params: Params) -> None:
+    wb = calc_workbook(params=params)
+
+    wb.save(
+        f"{params.file_slug}_{params.train1_arriving_pax}_{params.train2_arriving_pax}_{params.arr_time2 - params.arr_time1}s.xlsx"
+    )
+    wb.close()
+
+
 def main() -> None:
-    params = Params(
-        file_slug="platform3_pr_alt3_new",
+    params_p3120 = Params(
```

Comment:
Can you add the comments for what the `p*` names mean?

Reply:
they follow a pattern, its p<platform #><time in seconds> and recon means they account for penn reconstruction in the platform widths. i have added a comment to note this.